### PR TITLE
[DOCS] 7.14.1 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #969.

Preview [here](https://security-docs_987.docs-preview.app.elstc.co/guide/en/security/7.x/release-notes.html).

### CODE REVIEW

File: docs/release-notes.asciidoc [hunk 1]
```diff
@@ -13,6 +13,24 @@ This section summarizes the changes in each release.
...
 //== 7.14
 
+[discrete]
+[[release-notes-7.14.1]]
+=== 7.14.1
+
+[discrete]
+[[features-7.14.1]]
+==== Features
+The `securitySolution:defaultThreatIndex` advanced setting allows you to specify one or more threat intelligence index from which {es-sec} will collect threat match data. It controls features that query threat indexes, such as the Threat Intelligence view on the Overview page, and uses the `filebeat-*` index by default.
```
Comment (jmikell821) [inline docs/release-notes.asciidoc:23]:
```suggestion
The `securitySolution:defaultThreatIndex` advanced setting allows you to specify one or more threat intelligence indices from which {elastic-sec} will collect threat match data. It controls features that query threat indices, such as the Threat Intelligence view on the Overview page, and uses the `filebeat-*` index by default.
```

File: docs/release-notes.asciidoc [hunk 2]
```diff
@@ -13,6 +13,24 @@ This section summarizes the changes in each release.
...
+
+You can enable cross cluster search by adding threat indices from remote clusters. Filebeat indices are supported. Non-Filebeat indices require ECS-compliant threat field mappings to ensure threat matching works ({pull}108389[#108389]).
+
+[discrete]
+[[bug-fixes-7.14.1]]
+==== Bug fixes and enhancements
+* Fixes the AlienVault OTX event count on the Threat Intelligence view ({pull}108448[#108448]).
+* Fixes a configuration error in the `kibana.json` file ({pull}107637[#107637]).
+* Fixes UI errors that were caused by the rule `author` field not being migrated. ({pull}107230[#107230]).
```
Comment (jmikell821) [inline docs/release-notes.asciidoc:32]:
```suggestion
* Fixes UI errors that were caused by the rule `author` field not being migrated ({pull}107230[#107230]).
```

**Review by jmikell821 — COMMENTED** [review]
A few small nits, then OK to merge to `7.x`, then backport to `7.14` and `7.15`. If you run into any issues, contact Lisa.

File: docs/release-notes.asciidoc [hunk 3]
```diff
@@ -13,6 +13,24 @@ This section summarizes the changes in each release.
...
+[discrete]
+[[release-notes-7.14.1]]
+=== 7.14.1
+
+[discrete]
+[[features-7.14.1]]
+==== Features
+The `securitySolution:defaultThreatIndex` advanced setting allows you to specify one or more threat intelligence indices from which {elastic-sec} will collect threat match data. It controls features that query threat indices, such as the Threat Intelligence view on the Overview page, and uses the `filebeat-*` index by default.
+
+You can enable cross cluster search by adding threat indices from remote clusters. Filebeat indices are supported. Non-Filebeat indices require ECS-compliant threat field mappings to ensure threat matching works ({pull}108389[#108389]).
```
Comment (rylnd) [inline docs/release-notes.asciidoc:25]: Same note here as in #993 ; the `Filebeat indices are supported.` sentence raises more questions than it answers, IMO.

**Review by rylnd — CHANGES_REQUESTED** [review]
I think we just need a better description for the setting (#993) , and then this'll be good to go.

**Review by ecezalp — APPROVED** [review]
LGTM, adding some minor comments

> The setting controls features that query threat indices

I think we can say `configures` instead of `controls`

> such as the Threat Intelligence view on the Overview page and threat indicator match rules.

I think we can say `the Threat Intelligence view on the Overview page and the default indicator index values for indicator match rules.`

**Pull request merged**